ts/features/idpay/payment/screens/IDPayPaymentCodeScanScreen.tsx

```diff
@@ -1,36 +1,45 @@
 import { useIsFocused, useNavigation } from "@react-navigation/native";
 import React from "react";
 import { SafeAreaView, StatusBar, StyleSheet, View } from "react-native";
-import RNReactNativeHapticFeedback from "react-native-haptic-feedback";
 import LinearGradient from "react-native-linear-gradient";
 import { useSafeAreaInsets } from "react-native-safe-area-context";
 import CameraScanMarkerSVG from "../../../../../img/camera-scan-marker.svg";
 import { IOColors } from "../../../../components/core/variables/IOColors";
 import IconButton from "../../../../components/ui/IconButton";
+import { useOpenDeepLink } from "../../../../hooks/useOpenDeepLink";
 import I18n from "../../../../i18n";
 import {
   AppParamsList,
   IOStackNavigationProp
 } from "../../../../navigation/params/AppParamsList";
 import { IOBarcode, useIOBarcodeScanner } from "../components/Barcode";
+import { useIOBarcodeFileReader } from "../components/Barcode/useIOBarcodeFileReader";
 import { BottomTabNavigation } from "../components/BottomTabNavigation";
 import { CameraPermissionView } from "../components/CameraPermissionView";
-import { useOpenDeepLink } from "../../../../hooks/useOpenDeepLink";
+import { IDPayPaymentRoutes } from "../navigation/navigator";
 
 const IDPayPaymentCodeScanScreen = () => {
+  const navigation = useNavigation<IOStackNavigationProp<AppParamsList>>();
   const isFocused = useIsFocused();
   const openDeepLink = useOpenDeepLink();
 
-  const handleBarcodeScanned = (barcode: IOBarcode) => {
+  const handleBarcodeSuccess = (barcode: IOBarcode) => {
     if (barcode.type === "IDPAY") {
-      RNReactNativeHapticFeedback.trigger("notificationSuccess", {
-        enableVibrateFallback: true,
-        ignoreAndroidSystemSettings: false
-      });
       openDeepLink(barcode.authUrl);
+    } else {
+      alert("Unknown barcode :(");
     }
   };
 
+  const handleBarcodeError = () => {
+    alert("Invalid barcode :(");
```

Comment:
Same as above, if they are a placeholder, what do you think to temporary disable them or at least change the message since an error page of feedback is available?

Reply:
Yes they are placeholder. These should be replaced with a toast notification

Reply:
Corrected in https://github.com/pagopa/io-app/pull/4732/commits/e3b6ef770156e4093e91a9530cebe6bb833aecca